### ISSUE TITLE
Javascript.so is not a valid Win32 application

### ISSUE THREAD

**Issue by nappalm** [tree-sitter/tree-sitter#1470]
I got the following error after installing javascript : TSInstall javascript

Error after open js file
![image](https://user-images.githubusercontent.com/35016608/140344948-8f26fdd0-82e1-4e42-ad91-6a2c8090d6c4.png)

**Comment by ahelwer** [comment]
You probably want to file this but with the https://github.com/nvim-treesitter/nvim-treesitter repo.

**Comment by theHamsta** [comment]
and you could try one of the parsers compiled on our CI to see whether compiling with a different compiler would solve this problems. Anyways make sure you're using a 64bit compiler with a 64bit Neovim.

**Comment by LUCASFRANKINC** [comment]
Hello,
I came across this issue and all I did was run `:TSInstall javascript` , confirm reinstallation and now works perfectly. That is if you experience this in 2023. Also, ensure you  have the nvim-treesitter in your environment path if you have an error of the executable not being found. 

Hope it helps.

![image](https://github.com/tree-sitter/tree-sitter/assets/58594771/447efe1a-8244-43d4-a8e4-3eaa8607a0d8)
